services/mfccExtractor.py

```diff
@@ -0,0 +1,23 @@
+import librosa
+import numpy as np
+import helpers.librosaHelper as librosaHelper
+
+def getMfcc(librosaPath, intervals):
+  output = []
+
+  # retrieve librosa file
+  audio, sampleRate = librosaHelper.loadLibrosa(librosaPath)
+  
+  # iterate intervals and calculate mfccs 
+  for idx, (start, end) in enumerate(intervals):
```

Comment:
`for (start, end) in intervals:` should do it no?

Reply:
oh true